src/main.cpp

```diff
@@ -26,188 +26,226 @@
 
 using namespace std;
 
-using hclock=chrono::high_resolution_clock;
-using TIME = NDTime;
-
-
-/***** SETING INPUT PORTS FOR COUPLEDs *****/
-struct inp_control : public cadmium::in_port<Message_t>{};
-struct inp_1 : public cadmium::in_port<Message_t>{};
-struct inp_2 : public cadmium::in_port<Message_t>{};
-/***** SETING OUTPUT PORTS FOR COUPLEDs *****/
-struct outp_ack : public cadmium::out_port<Message_t>{};
-struct outp_1 : public cadmium::out_port<Message_t>{};
-struct outp_2 : public cadmium::out_port<Message_t>{};
-struct outp_pack : public cadmium::out_port<Message_t>{};
-
-
-/********************************************/
-/****** APPLICATION GENERATOR *******************/
-/********************************************/
-template<typename T>
-class ApplicationGen : public iestream_input<Message_t,T> {
-public:
-  ApplicationGen() = default;
-  ApplicationGen(const char* file_path) : iestream_input<Message_t,T>(file_path) {}
-};
-
-
-int main(int argc, char ** argv) {
-
-  if (argc < 2) {
-   cout << "you are using this program with wrong parameters. Te program should be invoked as follow:";
-   cout << argv[0] << " path to the input file " << endl;
-   return 1; 
-  }
-
-  auto start = hclock::now(); //to measure simulation execution time
-
-/*************** Loggers *******************/
-  static std::ofstream out_data("abp_output.txt");
-    struct oss_sink_provider{
-        static std::ostream& sink(){          
-            return out_data;
-        }
+    using hclock=chrono::high_resolution_clock;
```

Comment:
One suggestion is to add spaces before and after = sign to enhance code readability.

Reply:
Suggestion taken into account. Thank You